src/System.Device.Gpio/System/Device/Gpio/ExceptionHelper.cs

```diff
@@ -73,6 +73,9 @@ private static string GetResourceString(ExceptionResource resource, int errorCod
                 case ExceptionResource.NotListeningForEventError:
                     message = $"Attempted to remove a callback for a pin that is not listening for events.";
                     break;
+                case ExceptionResource.LibGpiodNotInstalled:
+                    message = $"Libgpiod driver not installed. More information on: https://git.kernel.org/pub/scm/libs/libgpiod/libgpiod.git/about/ ";
```

Comment:
@joshfree as you suggested updated to give more meaningful message when libgpiod not installed, but not sure about exact wording of the message...

Reply:
Looks fine, There is an extra space in the end though.
Alternative: "Libgpiod driver not present. Kernel version X.Y.Z required."

Reply:
Thanks, not sure why you added "Kernel version X.Y.Z required", libgpiod still need to be installed manually even kernel version was above 4.8

Reply:
I see, I thought it was included in the kernel by default not a separate kernel module - makes sense in that case